packages/vscode-xml-tools-client/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "vscode-xml-tools-client",
```

Comment:
This is the name of the extension, it should be shorter for consumption in VSCode Marketplace.
- which means name here may not be equal to the folder name